src/Vehicle/Vehicle.cc

```diff
@@ -789,7 +789,7 @@ void Vehicle::_handleHilActuatorControls(mavlink_message_t &message)
 
 void Vehicle::_handleCommandAck(mavlink_message_t& message)
 {
-    bool showError = true;
+    bool showError = false;
```

Comment:
Make sense. I could see how this could happen when it retries. If the vehicle actually got both commands it will correctly ack twice.